include/grpcpp/security/tls_certificate_verifier.h

```diff
@@ -62,10 +62,12 @@ class TlsCustomVerificationCheckRequest {
   grpc::string_ref peer_cert() const;
   grpc::string_ref peer_cert_full_chain() const;
   grpc::string_ref common_name() const;
+  grpc::string_ref verified_root_cert_subject() const;
```

Comment:
We need to be explicit about the contract/format for this API: for example,

* If verification fails or the peer cert is self-signed, this will be the empty string.
* If verification is successful, it is a comma-separated list, where the entries are of the form "FIELD_ABBREVIATION=string" and give a list of supported field abbreviations.

We also need tests to ensure that this contract is adhered to.

Reply:
SGTM! And please add unit tests somewhere that enforce this.

Reply:
Working on more unit tests for this, particularly the self-signed cert scenario.
In terms of listing all support field abbreviations, I hesitate to create a listing of them here - it should be a list of anything that is valid for the subject name of a certificate, so the best place to get that information is the user's certificate, or an RFC

Reply:
Will add